### ISSUE TITLE
enhancement suggestion: Desktop Notification for records assigned to current user

### ISSUE THREAD

**Issue by vjaro** [salesagility/SuiteCRM#749]
This will allow the user to be notified immediately of the records assigned to him/her without having to login to their email client/provider. On the User advanced settings we can have a list of modules that can be toggled to enable this feature.

![alerts](https://cloud.githubusercontent.com/assets/3088198/11553744/abd6a1ac-99cc-11e5-8903-805df838fd64.png)

**Comment by gunnicom** [comment]
+1

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [enhancement suggestion: Desktop Notification for records assigned to current user](https://trello.com/c/xbZYLLcV/130-enhancement-suggestion-desktop-notification-for-records-assigned-to-current-user)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
